### PR TITLE
{tools,vis}[fosscuda/2020b] PyTorch3D v0.4.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

torchvision is in ~#13224~

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bask-pg0308u26a - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/097019fe67f35a24fc09459a143f06a8 for a full test report.

File: easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('tqdm', '4.56.2'),
+    ('PyTorch', local_pytorch_version),
+    ('imageio', '2.9.0'),
+    ('IPython', '7.18.1'),
+    ('torchvision', '0.9.1', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb:20]: @branfosj This should be lowered to 0.8.2 to stick to the single-dependency-version per easyconfigs generation rule?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb:20]: Yes, which will also put it against Pytorch 1.7.1. Oddly, that is what we have installed locally, but I must have bumped the version when pushing upstream.

Comment (branfosj) [reply easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb]: fixed in bd3aa47

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0212u17a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), 1 x NVIDIA Tesla P100-PCIE-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/5381a4fa008dfef9e6bce56f51a4c125 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3901.accelgor.os - Linux RHEL 8.4, x86_64, AMD EPYC 7413 24-Core Processor (zen2), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 470.82.01, Python 3.6.8
See https://gist.github.com/e59d93e40fb7f30649528a369a8ca020 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**